tests/helper/odo_utils.go

```diff
@@ -82,9 +82,26 @@ func CreateRandProject() string {
 	return projectName
 }
 
+// CreateRandNamespace create new project with random name in kubernetes cluster (10 letters)
+func CreateRandNamespace(context string) string {
```

Comment:
This is cool!!!, @prietyc123 Can we move those methods to different packages. Actually it will be easy to maintain only Kubernetes specific utils in later stage. 

For example : ```kubernetes_utils.go```

Reply:
> This is cool!!!, @prietyc123 Can we move those methods to different packages. Actually it will be easy to maintain only Kubernetes specific utils in later stage.

I think we can do that 👍